### PR TITLE
Refactor Cordova Docs

### PR DESCRIPTION
**Pull request by PeloWriter**
The work for this PR is tracked [here](https://github.com/getsentry/sentry-docs/issues/3285).

Thus far, 
- the common content is now being used for Getting Started
- the feature list is a combination of Flutter and React-Native features - @jennmueng, please review closely
- Debug Information file created ([later work](https://github.com/getsentry/sentry-docs/issues/3299) across the mobile SDKs will make this common, so this is a short-term solution)
- Created a Troubleshooting page specific to Cordova for the script related to iTunes Connect
- Performance content is not yet enabled, but can easily be - just let me know when @jennmueng 
- the section in the Getting Started for iOS specifics is fine, but are we missing a similar set of content for Android (as we have [for React-Native](https://docs.sentry.io/platforms/react-native/#android-specifics))?
- The verification step of Getting Started is identical to JavaScript, I can remove the Cordova example since it will simply fall back to JS? @jennmueng

### CODE REVIEW

**Comment by jennmueng** [comment]
Hey @PeloWriter sorry for the late reply!
- On the features list, performance monitoring only creates transactions automatically given you enable it with `@sentry/tracing` and `BrowserTracing`. Everything else in the features list looks good. Maybe also include that all of iOS, Android, and Browser is supported (we didn't officially support browser prior to 1.0.0)
- Performance content is the same as browser, you would just directly use `BrowserTracing` from `@sentry/tracing` directly like on JS browser. Feel free to enable it as we're about to release 1.0.0 GA that supports it.
- For Android, we do not have automatic source map uploads or any build scripts for that matter. So currently only iOS is correct until we add them.
- For the verification step, let's copy over React Native's, as `nativeCrash` is now added in 1.0.0

**Comment by PeloWriter** [comment]
> On the features list, performance monitoring only creates transactions automatically given you enable it with @sentry/tracing and BrowserTracing. Everything else in the features list looks good. Maybe also include that all of iOS, Android, and Browser is supported (we didn't officially support browser prior to 1.0.0)
Added support for iOS and Android (JS already noted), removed the term "automatically" 
> Performance content is the same as browser, you would just directly use BrowserTracing from @sentry/tracing directly like on JS browser. Feel free to enable it as we're about to release 1.0.0 GA that supports it.
> For Android, we do not have automatic source map uploads or any build scripts for that matter. So currently only iOS is correct until we add them.
For the Debug Information page, we're only providing a run script for Xcode, so we've got a spot when Android support is available
> For the verification step, let's copy over React Native's, as nativeCrash is now added in 1.0.0
Done.

**Comment by marandaneto** [comment]
@PeloWriter @jennmueng related to:

> For Android, we do not have automatic source map uploads or any build scripts for that matter. So currently only iOS is correct until we add them.

either we write down this limitation and redirect the users to the sentry-cli page (how to do it manually) or we write down it's not supported, just omitting sounds suboptimal and pretty we'll get asked because of that.

File: src/includes/getting-started-primer/javascript.cordova.mdx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+- Automatic Native Crash [Error Tracking](/product/issues/) using both [iOS](/platforms/apple/guides/ios/) and [Android](/platforms/android/)
+- Offline storage of events
+- Events [enriched](/platforms/javascript/guides/cordova/enriching-events/context/) with device data
+- [Breadcrumbs](/platforms/javascript/guides/cordova/enriching-events/breadcrumbs/) created for location changes
+- On-device symbolication for JavaScript (in `Debug` mode)
+- [Ionic support](/platforms/javascript/guides/cordova/ionic/)
+- Under the hood the SDK relies on our [JavaScript SDK](/platforms/javascript/), which makes all functions available for JavaScript also available in this SDK including:
+    - [Performance Monitoring](/product/performance/) to create transactions
+    - [Release Health](/product/releases/health/) to track crash free users and sessions
```
Comment (marandaneto) [inline src/includes/getting-started-primer/javascript.cordova.mdx:19]: release health is supported thru the Native SDK and not the JavaScript SDK.
should we move it under `Automatic Native Crash`?

File: src/platforms/javascript/guides/cordova/upload-debug.mdx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+title: Debug Information
+sidebar_order: 2
+description: "Learn more about uploading debug symbols for when building your app."
```
Comment (marandaneto) [inline src/platforms/javascript/guides/cordova/upload-debug.mdx:4]: @jennmueng what's about source maps? does it work automatically for Android and iOS?

File: src/platforms/javascript/guides/cordova/troubleshooting.mdx [hunk 3]
```diff
@@ -0,0 +1,39 @@
+---
+title: Troubleshooting
+description: "Troubleshoot and resolve issues when submitting to iTunes Connect."
+sidebar_order: 1000
+---
+
+If you need help solving issues with your Sentry Cordova integration, you can read the edge cases documented here. If you need additional help, you can view our forums, and customers on a paid plan may also contact support.
+
+## Error When Submitting to iTunes Connect
```
Comment (jennmueng) [inline src/platforms/javascript/guides/cordova/troubleshooting.mdx:9]:
```suggestion
## Error When Submitting to App Store Connect
```

File: src/platforms/javascript/guides/cordova/troubleshooting.mdx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+If you are receiving an error while submitting your build to iTunes Connect, this script is likely missing. In addition, re-adding the plugin can resolve the error.
```
Comment (jennmueng) [inline src/platforms/javascript/guides/cordova/troubleshooting.mdx:11]:
```suggestion
If you are receiving an error while submitting your build to App Store Connect, this script is likely missing. In addition, re-adding the plugin can resolve the error.
```

File: src/platforms/javascript/guides/cordova/upload-debug.mdx (hunk 2 above)
Comment (jennmueng) [reply src/platforms/javascript/guides/cordova/upload-debug.mdx:4]: Yes it does, it actually automatically uploads **js**-only sourcemaps on at build time on `cordova build`. Maybe we can create a Cordova-specific source maps section @PeloWriter as the current sourcemaps section for JS is geared towards browser and mostly not relevant to Cordova.

File: src/includes/getting-started-config/javascript.cordova.mdx [hunk 5]
```diff
@@ -7,8 +7,16 @@ onDeviceReady: function() {
...
 
 ```xml {filename:config.xml}
 <access origin="sentry.io" />
 ```
+
+### Debug Information
+
+If you want to run the debug information (also known as debug symbols) upload when building your app, you can use the scripts provided in [Debug Information](/platforms/javascript/guides/cordova/upload-debug/).
+
+### Script to Submit to iTunes Connect
```
Comment (jennmueng) [inline src/includes/getting-started-config/javascript.cordova.mdx:20]:
```suggestion
### Script to Submit to App Store Connect
```

**Review by jennmueng — COMMENTED** [review]
Thanks Fiona! Just a couple comments, also sorry for the confusion that although there are no build scripts for Android debug symbols there actually is a JS-level one for release versions and source maps that gets called regardless of the platform.

**Comment by PeloWriter** [comment]
I've added these two statements to the Debug Files page:
"If you are using our Cordova SDK, we do not currently support automatic source map uploads or build scripts for Android. You must upload files manually [using our CLI](/product/cli/dif/#uploading-files)."

"JS Sourcemaps are automatically uploaded at build time on `cordova build`."

I can also add those notes to the start of the JS core source maps content, on the "Uploading Source Maps" page, if that is sufficient. But given @jennmueng's note that much of the source maps content is not relevant to Cordova, I'm wondering if we should turn off the display of it and simply have a Cordova-specific page?

**Comment by PeloWriter** [comment]
Also, we still need to verify the list of options for 1.0.0.

**Comment by jennmueng** [comment]
@PeloWriter Yes let's turn the current one off and have a separate source maps page for Cordova. 

Regarding the features list, we also have breadcrumbs for outgoing http request with XHR and Fetch, console logs, and HTML touch events on top of the location breadcrumbs already mentioned. Otherwise everything looks good!

File: src/platforms/javascript/guides/cordova/upload-debug.mdx [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+title: Debug Information
+sidebar_order: 2
+description: "Learn more about uploading debug symbols when building your app."
+---
+
+Debug symbols provide information that Sentry displays on the Issue Details page to help you triage an issue.
+
+<Note>
+
+We do not currently support automatic source map uploads or build scripts for Android. You must upload files manually [using our CLI](/product/cli/dif/#uploading-files)."
```
Comment (jennmueng) [inline src/platforms/javascript/guides/cordova/upload-debug.mdx:11]:
```suggestion
We do not currently support non-javascript automatic source map uploads or build scripts for Android. You must upload files manually [using our CLI](/product/cli/dif/#uploading-files)."
```
Maybe we can make it clear here that this is referring only to non-javascript debug files?

**Review by jennmueng — APPROVED** [review]

**Pull request merged**